docs/resources/azure_bastion_hosts_resource.md

```diff
@@ -0,0 +1,100 @@
+---
+title: About the azure_bastion_hosts_resource Resource
+platform: azure
+---
+
+# azure_bastion_hosts_resource
+
+Use the `azure_bastion_hosts_resource` InSpec audit resource to test properties related to a bastion hosts resource.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group` and bastion hosts resource `name` or the `resource_id` must be given as a parameter.
+```ruby
+describe azure_bastion_hosts_resource(resource_group: 'MyResourceGroup', name: 'bastion_name') do
+  it { should exist }
+end
+```
+## Parameters
+
+| Name                           | Description                                                                      |
+|--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in. `MyResourceGroup`     |
+| bastionHostName                           | Name of the virtual network to test. `MyVNetwork`                                 |
```

Comment:
in source code you use `name` as required parameter

Reply:
Yes.Corrected both places.